### ISSUE TITLE
Parameter bounds propagation in xFitModel summing

### ISSUE THREAD

**Issue by albertomanfreda** [lucabaldini/ixpeobssim#658]
Currently, summing two classes derived by xFitModelBase does not preserve the default parameter bounds. 
This can be fixed by the following change:

```
@@ -328,7 +328,10 @@ class xFitModelBase:
             PARAMETER_DEFAULT_VALUES = m1.PARAMETER_DEFAULT_VALUES + \
                                        m2.PARAMETER_DEFAULT_VALUES
             DEFAULT_PLOTTING_RANGE = (xmin, xmax)
-            PARAMETER_DEFAULT_BOUNDS = (-numpy.inf, numpy.inf)
+            PARAMETER_DEFAULT_BOUNDS = (m1.PARAMETER_DEFAULT_BOUNDS[0] +
+                                        m2.PARAMETER_DEFAULT_BOUNDS[0],
+                                        m1.PARAMETER_DEFAULT_BOUNDS[1] +
+                                        m2.PARAMETER_DEFAULT_BOUNDS[1])
```

However, this fails for models that do not specify their bounds, as the default bounds are a tuple of values, rather than a tuple of lists.
So the following change is needed as well:

```
@@ -84,7 +84,7 @@ class xFitModelBase:
 
     PARAMETER_NAMES = ()
     PARAMETER_DEFAULT_VALUES = ()
-    PARAMETER_DEFAULT_BOUNDS = (-numpy.inf, numpy.inf)
+    PARAMETER_DEFAULT_BOUNDS = ([-numpy.inf], [numpy.inf])
     DEFAULT_PLOTTING_RANGE = (0., 1.)
     DEFAULT_STAT_BOX_POSITION = 'upper right'
```

**Comment by albertomanfreda** [comment]
And eventually I ended up doing something different and moving all the logic in the __sum__ method.

**Comment by albertomanfreda** [comment]
Proposed implementation https://github.com/lucabaldini/ixpeobssim/pull/660

**Comment by lucabaldini** [comment]
This is now merged into the main, waiting for a new tag.

**Comment by lucabaldini** [comment]
Shipped in 29.4.0
